source/ProtonPack/Serial.h

```diff
@@ -3479,28 +3479,40 @@ void handleWandCommand(uint16_t i_command, uint16_t i_value) {
       resetContinuousSmoke();
     break;
 
-    case W_BARREL_LEDS_48:
+    case W_BARREL_LEDS_5:
       stopEffect(S_VOICE_BARREL_LED_5);
+      //stopEffect(S_VOICE_BARREL_LED_29);
       stopEffect(S_VOICE_BARREL_LED_48);
       stopEffect(S_VOICE_BARREL_LED_60);
 
-      playEffect(S_VOICE_BARREL_LED_48);
+      playEffect(S_VOICE_BARREL_LED_5);
     break;
 
-    case W_BARREL_LEDS_60:
+    case W_BARREL_LEDS_29:
```

Comment:
@nomakewan Same case here, this shouldn't be needed because this isn't a selectable option for users, and is only for internal testing.